src/signals/IncidentMap/components/IncidentMap/IncidentMap.tsx

```diff
@@ -55,11 +81,59 @@ export const IncidentMap = () => {
     }
   }, [error, isSuccess])
 
+  useEffect(() => {
+    if (!map || !pinMarker || !coordinates) return
```

Comment:
for such a small piece of logic, consider changing it to 

```
if (map && pinMarker && coordinates) {
    pinMarker.setLatLng(coordinates)
}
```

this is more readable

Reply:
done